crates/router/src/db/fraud_check.rs

```diff
@@ -116,33 +116,3 @@ impl FraudCheckInterface for MockDb {
         Err(errors::StorageError::MockDbError)?
     }
 }
-
-#[cfg(feature = "kafka_events")]
-#[async_trait::async_trait]
-impl FraudCheckInterface for super::KafkaStore {
```

Comment:
why is this removed?

Reply:
`#[cfg(feature = "kafka_events")]`
It used to be a feature flag, but now it is not being used.

Reply:
> why is this removed?

It used to be a feature flag, but now it is not being used.